src/test/unit/controls/Graph/GraphGenerate-spec.js

```diff
@@ -507,10 +507,39 @@ describe("Graph - Generate", () => {
                 axisDefault.x.label
             );
         });
-        it("Creates label y axis when text is present", () => {
+        it("Creates label y axis when y2-axis is false and when text is present", () => {
             graph.destroy();
             graph = new Graph(getAxes(axisDefault));
 
+            const yLabelElement = fetchElementByClass(styles.axisLabelY);
+            const translate = getSVGAnimatedTransformList(
+                yLabelElement.getAttribute("transform")
+            ).translate;
+            expect(yLabelElement.nodeName).toBe("g");
+            expect(yLabelElement.querySelector("tspan").textContent).toBe(
+                axisDefault.y.label
+            );
+            expect(yLabelElement.getAttribute("transform")).toContain(
+                "rotate(-90)"
+            );
+            expect(toNumber(translate[0], 10)).toBeCloserTo(20);
+            expect(toNumber(translate[1], 10)).toBeCloserTo(115);
+        });
+        it("Creates label y axis when y2-axis is true and text is present", () => {
```

Comment:
nit: It can be updated like below
```js
describe("when y2-axis is true and y-axis label is present", () => {
     it("should create y-axis label and not reduce the gap between y-axis and y-axis label", () => {
    });
});
``

Reply:
Well that sounds good. But in order to maintain the consistency with previous test cases i just wrote it this way.